utils/utils.go

```diff
@@ -139,15 +139,20 @@ func FillSlice(value byte, data []byte) {
 	}
 }
 
+// ZeroizeBytes wipes a byte slice from memory, filling it with zeros.
+func ZeroizeBytes(data []byte) {
+	FillSlice(0, data)
```

Comment:
This should be faster, as mentioned [here](https://github.com/cossacklabs/acra/pull/409#issuecomment-691951455):
```suggestion
	for i := range data {
		data[i] = 0
	}
```
Go compiler may be too dumb to inline call to `FillSlice()` and use `runtime.memclrNoHeapPointers` on `data`

Reply:
Okay, you win. I feel that this is mostly pointless optimization, but for the sake of resolving this argument once and for all, I need to do something.

I'll replace new uses of `FillSlice` with inlined loop in this PR. Then, after this PR is merged, there will be a separate PR replacing all remaining uses of `FillSlice(0, ...)` with appropriate `Zeroize...()` call. And then I'll remove `FillSlice` entirely from the code base since that's the only use it has.